scipy/signal/lti_conversion.py

```diff
@@ -459,6 +465,25 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
         cd = c
         dd = d
 
+    elif method == 'foh':
+        # Build an exponential matrix similar to 'zoh' method
+        em_upper = linalg.block_diag(np.block([a, b]) * dt, np.eye(b.shape[1]))
+        em_lower = zeros((b.shape[1], a.shape[0] + 2 * b.shape[1]))
+        em = np.block([[em_upper], [em_lower]])
+
+        ms = linalg.expm(em)
+
+        # Get the three blocks from upper rows
+        ms11 = ms[:a.shape[0], :a.shape[0]]
+        ms12 = ms[:a.shape[0], a.shape[0]:a.shape[0] + b.shape[1]]
+        ms13 = ms[:a.shape[0], a.shape[0] + b.shape[1]:]
+
+        ad = ms11
+        bd = ms12 - ms13 + ms11.dot(ms13)
```

Comment:
In v1.3.0 and later we won't be supporting <Python 3.5 hence you can use `@` symbol instead of `dot`.